src/extension/common/components/copyButton.re

```diff
@@ -1,4 +1,10 @@
-let createElement ::label="copy" ::text ::onCopy ::style=? children::_ () =>
-  <CopyToClipboard text onCopy>
-    <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
-  </CopyToClipboard>;
+
+let component = ReasonReact.statelessComponent "CopyButton";
+let make ::label="copy" ::text ::onCopy ::style=? _ => {
+  ...component,
+
+  render: fun _ _ =>
+    <CopyToClipboard text onCopy>
+      <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
+    </CopyToClipboard>
+};
```

Comment:
Using a function directly kind of breaks the abstraction. If we later need to change it into a component, to add local state or something, we'll have to change all the call sites. And of course it kind of defeats the purpose of JSX if you can't trust components to look like components. So it's not the end of the world, but it's not good...